File: app/src/main/java/com/example/aad_team_35_animation_challenge/SplashScreenActivity.java
package com.example.aad_team_35_animation_challenge;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;

import com.example.aad_team_35_animation_challenge.auth.AuthActivity;
import com.example.aad_team_35_animation_challenge.onboarding.OnBoardingActivity;


public class SplashScreenActivity extends AppCompatActivity {
    ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        imageView = findViewById(R.id.splashImage);
        startAnim();

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                startActivity(
                        new Intent(SplashScreenActivity.this, OnBoardingActivity.class)
                );

                finish();
            }
        }, 1500);
    }

    private void startAnim() {
        Animation animation = AnimationUtils.loadAnimation(getBaseContext(), R.anim.onboard_anim);
        imageView.startAnimation(animation);


    }
}

File: app/src/main/java/com/example/aad_team_35_animation_challenge/SplashScreenActivity.kt
package com.example.aad_team_35_animation_challenge

import android.content.Intent
import android.os.Bundle
import android.os.Handler

import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import com.example.aad_team_35_animation_challenge.app.QuizMoApp.Companion.animate

import com.example.aad_team_35_animation_challenge.databinding.ActivitySplashScreenBinding
import com.example.aad_team_35_animation_challenge.onboarding.OnBoardingActivity


class SplashScreenActivity : AppCompatActivity() {

    private var mBinding: ActivitySplashScreenBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_splash_screen)

        animate(mBinding!!.txtAppName, R.animator.alpha)
        animate(mBinding!!.imgSplash, R.animator.scale_animation)

        Handler().postDelayed({
            startActivity(
                    Intent(this@SplashScreenActivity, OnBoardingActivity::class.java)
            )

            finish()
        }, 5000)
    }
}
